packages/larva-patterns/objects/o-figure/o-figure.twig

```diff
@@ -2,42 +2,33 @@
 	<a href="{{ o_figure_permalink_url }}" class="o-figure__link {{ o_figure_link_classes }}">
 {% endif %}
 
-
 <figure class="o-figure {{ modifier_class }} {{ o_figure_classes }} lrv-u-max-width-100p" style="{{ o_figure_width_attr }}">
 
-	{% if o_figure_screen_reader_text %}
-		<span class="lrv-a-screen-reader-only">{{ o_figure_screen_reader_text }}</span>
-	{% endif %}
-
 	{% if o_figure_crop_class %}
 		<div class="{{ o_figure_crop_class }}">
-	{% else %}
-		<div style="{{ o_figure_crop_style_attr }}">
+	{% endif %}
+
+	{% if o_figure_crop_style_attr %}
+		<div class="lrv-a-crop-4x3" style="{{ o_figure_crop_style_attr }}">
```

Comment:
@laras126 Can you explain this change?

Reply:
This is for the inline CSS for adjusting the padding according the image dimensions in the_content – I will update the naming so it is more clear.

Reply:
@yahilmadakiya updated o-figure to pull in c-lazy-image as intended. Good catch!